ProcessOptimizer/space/constraints.py

```diff
@@ -476,6 +573,53 @@ def __eq__(self, other):
         else:
             return False
 
+class Sum_equals():
```

Comment:
```suggestion
class SumEquals():
```
To follow the usual standard for class names.